index.d.ts

```diff
@@ -245,7 +394,7 @@ export interface DailyCallStaticUtils {
 
 export interface DailyCall {
   iframe(): HTMLIFrameElement | null;
-  join(properties?: DailyCallOptions): Promise<DailyParticipant[] | void>;
+  join(properties?: DailyCallOptions): Promise<DailyParticipantsObject | void>;
```

Comment:
Great catch on resolving this promise with an object rather than array, @hueter! You'll see I added the `void` back in. It was there to indicate that the promise could simply resolve with no arguments whatsoever (which happens in the case that we were already in the meeting and so the join() was a no-op). Calling code would then have to guard against `undefined` (not `null`).

Reply:
ah okay I see